contextPlus.js

```diff
@@ -27,13 +27,34 @@ const contextMenuContainers = {
       parentId
     });
 
+    const colors = {
+      blue: '00a7e0',
+      turquoise: '01bdad',
+      green: '7dc14c',
+      yellow: 'ffcb00',
+      orange: 'ff9216',
+      red: 'd92215',
+      pink: 'ee5195',
+      purple: '7a2f7a'
+    };
+
     contextualIdentities.forEach(context => {
-      browser.contextMenus.create({
-        type: "normal",
-        title: context.name,
-        id: `contextPlus-${context.name}`,
-        parentId
+
+      fetch("icons/usercontext-"+context.icon+".svg").then( response => response.text()).then( svg => {
```

Comment:
Can we use template literals instead of the `+` operator for constructing strings?

```javascript
`icons/usercontext-${context.icon}.svg`
```

Same on lines 47 and 52.